src/config/bluetooth.md

```diff
@@ -0,0 +1,84 @@
+# Bluetooth
+
+Ensure the bluetooth controller is not blocked. Use `rfkill` to check whether
+there are any blocks and to remove soft blocks. If there is a hard block, there
+is likely either a physical hardware switch or an option in the BIOS to enable
+the bluetooth controller.
+
+```
+$ rfkill
+ID TYPE     DEVICE      SOFT      HARD
+0 wlan      phy0   unblocked unblocked
+1 bluetooth hci0     blocked unblocked
+
+# rfkill unblock bluetooth
+```
+
+## Installation
+
+Install the `bluez` package.
+
+```
+# xbps-install -S bluez
+```
+
+Enable the `bluetoothd` and `dbus` services.
+
+```
+# ln -s /etc/sv/bluetoothd /var/service
+# ln -s /etc/sv/dbus /var/service
+```
+
+Add your user to the `lp` group and restart the `dbus` service.
```

Comment:
I could also say either reboot entirely or restart dbus, and if you choose to restart dbus it might have this unintended side-effect? I'll write that out so you can see how it might look